package-lock.json

```diff
@@ -3280,6 +3279,12 @@
         "object-assign": "4.1.1"
       }
     },
+    "grunt-search": {
+      "version": "0.1.8",
+      "resolved": "https://registry.npmjs.org/grunt-search/-/grunt-search-0.1.8.tgz",
+      "integrity": "sha1-9K1kQLr49d+122p/KSeUDNJ9Vec=",
+      "dev": true
+    },
```

Comment:
Not sure what this is about. Side effect of building the file?

Reply:
package-lock.json is an effect of building - but it's also necessary for determining dependencies and what can be updated. It's safe to remain, and the build will work fine without it. It could be separated off into its own branch but that would make some of the process manual. Better not to.